include/libtorrent/file.hpp

```diff
@@ -187,6 +187,14 @@ namespace libtorrent
 // internal export should be used at unit tests only
 	TORRENT_EXTRA_EXPORT native_path_string convert_to_native_path_string(std::string const& path);
 
+// internal export should be used at unit tests only
+	TORRENT_EXTRA_EXPORT std::string convert_from_native_path(char const* s);
```

Comment:
I was under the impression that mingw doesn't have ``std::wstring`` and that ``TORRENT_USE_WSTRING`` would not be defined in that case. But maybe I'm mixing it up with cygwin.